install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -69,88 +69,69 @@ Because the Aggregator PV is relatively small, the least expensive performance g
 
 ```yaml
 kubecostAggregator:
-  # deployMethod determines how Aggregator is deployed. Current options are
-  # "singlepod" (within cost-analyzer Pod) "statefulset" (separate
-  # StatefulSet), and "disabled". Only use "disabled" if this is a secondary
-  # Federated ETL cluster which does not need to answer queries.
-  deployMethod: singlepod
-
-  # fullImageName overrides the default image construction logic. The exact
-  # image provided (registry, image, tag) will be used for aggregator.
-  # fullImageName:
-  imagePullPolicy: IfNotPresent
-
-  # For legacy configuration support, `enabled: true` overrides deployMethod
-  # and causes `deployMethod: "statefulset"`
-  enabled: false
-
-  # Replicas sets the number of Aggregator replicas. It only has an effect if
-  # `deployMethod: "statefulset"`
-  replicas: 1
```

Comment:
These are default values which don't need to be included in the "optimization" section of the doc.